File: Assets/VirtualCity/ProtoDefine/ReqCreateCommentsMessage.cs
using System.Collections.Generic;
using ProtoBuf;
namespace ProtoDefine {
[ProtoContract]


public class ReqCreateCommentsMessage {
    /**评论信息*/
[ProtoMember(1)]
    public Comment comment;
[ProtoMember(2)]
    public string orderNo;

    public string getOrderNo() {
        return orderNo;
    }

    public void setOrderNo(string orderNo) {
        this.orderNo = orderNo;
    }

    public Comment getComment() {
        return comment;
    }

    public void setComment(Comment comment) {
        this.comment = comment;
    }
}
}

File: Assets/VirtualCity/ProtoDefine/Java/mall/ReqCreateCommentsMessage.java
package com.kingston.jforgame.server.game.mall.message;

import com.baidu.bjf.remoting.protobuf.FieldType;
import com.baidu.bjf.remoting.protobuf.annotation.Protobuf;
import com.kingston.jforgame.server.game.Modules;
import com.kingston.jforgame.server.game.entity.mall.Comment;
import com.kingston.jforgame.server.game.mall.MallDataPool;
import com.kingston.jforgame.socket.annotation.MessageMeta;
import com.kingston.jforgame.socket.message.Message;

@MessageMeta(module = Modules.SHOPPING,cmd = MallDataPool.REQ_CREATECOMMENT)
public class ReqCreateCommentsMessage extends Message {
    /**评论信息*/
    @Protobuf(order = 1,fieldType = FieldType.OBJECT)
    private Comment comment;
    @Protobuf(order = 2)
    private String orderNo;

    public String getOrderNo() {
        return orderNo;
    }

    public void setOrderNo(String orderNo) {
        this.orderNo = orderNo;
    }

    public Comment getComment() {
        return comment;
    }

    public void setComment(Comment comment) {
        this.comment = comment;
    }
}
